community/SDK-GOVERNANCE.md

```diff
@@ -29,9 +29,9 @@ projects. The purpose of these requirements are to demonstrate the person's
 expertise and regular commitment to the project - not simply to achieve a
 certain level of activity.
 
-Each sdk project MAY define in the `CONTRIBUTING.md` (available at the root of
-the Github Repository) stricter requirements, in order to meet the community
-demands, e.g. _we require that the contributor submitted at least 20 PRs_.
+Each sdk project may define in their `CONTRIBUTING.md` file stricter
+requirements, in order to meet the community demands, e.g. _we require that
+the contributor submitted at least 20 PRs_.
```

Comment:
Just wanted to make it clear that we're talking about the CONTRIBUTING in the sdk repo and not the main repo